.coffeelintignore

```diff
@@ -1,2 +0,0 @@
-iconsets
-node_modules
```

Comment:
Then this is probably a "bug" in the `build-package.sh` file. One way to check if linting is run is to make a temporary bug.

p.s. I'm all for squashing commits when the situation seems right (which it does in this case), but only when everything is finished and not within a review process :)